src/zmap.c

```diff
@@ -230,12 +234,12 @@ static void start_zmap(void)
 	}
 	log_debug("zmap", "%d sender threads spawned", zconf.senders);
 
-	monitor_init();
-	mon_start_arg_t *mon_arg = xmalloc(sizeof(mon_start_arg_t));
-	mon_arg->it = it;
-	mon_arg->recv_ready_mutex = &recv_ready_mutex;
-	mon_arg->cpu = zconf.pin_cores[cpu % zconf.pin_cores_len];
-	{
+	if (!zconf.dryrun) {
```

Comment:
monitor runs even without --quiet. Later on it decides if it should stick around.